### PR TITLE
Jupyter for r

### PR DESCRIPTION
**Pull request by tpfau**
Update instructions to install R versions for Jupyter.

### CODE REVIEW

File: triton/apps/jupyter.rst [hunk 1]
```diff
@@ -259,6 +259,78 @@ modules you need::
   # Use envkernel to re-write, loading the R modules.
   envkernel lmod --user --kernel-template=NAME --name=NAME r-irkernel/1.1-python3
 
+
+Installing a different R version as a kernel
+-------------------------------------------
```
Comment (rkdarst) [inline triton/apps/jupyter.rst:264]:
```suggestion
--------------------------------------------
```

File: triton/apps/jupyter.rst [hunk 2]
```diff
@@ -259,6 +259,78 @@ modules you need::
...
+           # his will use the latest R version on conda-forge. If you need a specific version you can specify it
+           # as r-essentials=X.X.X, where X.X.X is your required R version number
+           mamba create -n NAME -c conda-forge r-essentials r-irkernel 
+
+        The next steps are the same as building a Kernel, except for activating the environment instead of 
+        loading the r-irkernel module, since this module depends on the R version.
+        the ``displayname`` will be what will be displayed on jupyter ::
+        
+          # Use Rscript to install jupyter kernel, you need the environment for this.
+          module load jupyterhub/live
```
[the remark this answers is not in the document]
Comment (tpfau) [reply triton/apps/jupyter.rst:291]: It is, IRkernel requires it. Probably to link to jupyter.

File: triton/apps/jupyter.rst [hunk 3]
```diff
@@ -259,6 +259,78 @@ modules you need::
...
+          source activate NAME
+          Rscript -e "library(IRkernel); IRkernel::installspec(name='NAME', displayname='YOUR R Version')"
+          conda deactivate NAME
+
+          # For R versions before 4, you need to install the kernel. After version 4 IRkernel automatically installs it.
+          envkernel lmod --user --kernel-template=NAME --name=NAME
```
[the remark this answers is not in the document]
Comment (tpfau) [reply triton/apps/jupyter.rst:297]: As the module is needed before, I'll just leave it in there,

File: triton/apps/jupyter.rst [hunk 4]
```diff
@@ -259,6 +259,78 @@ modules you need::
...
+         install.packages('IRkernel') 
+         # exit R again
+
+       Create the installation specs using Rscript and IRKernel. Select a ``NAME`` for the environment specification
+       that can be used to install it. The
+       Next install the jupyter kernel. Here you need to select the ``NAME`` given before. 
+       The NAME is what is will be referred to for installation, while ``DISPLAYNAME`` will be displayed in jupyter::
+
+         # Use Rscript to install jupyter kernel, you need the environment for this.
+         module load jupyterhub/live
```
[the remark this answers is not in the document]
Comment (tpfau) [reply triton/apps/jupyter.rst:321]: see above

File: triton/apps/jupyter.rst [hunk 5]
```diff
@@ -259,6 +259,78 @@ modules you need::
...
+        loading the r-irkernel module, since this module depends on the R version.
+        the ``displayname`` will be what will be displayed on jupyter ::
+        
+          # Use Rscript to install jupyter kernel, you need the environment for this.
+          module load jupyterhub/live
+          source activate NAME
+          Rscript -e "library(IRkernel); IRkernel::installspec(name='NAME', displayname='YOUR R Version')"
+          conda deactivate NAME
+
+          # For R versions before 4, you need to install the kernel. After version 4 IRkernel automatically installs it.
```
[the remark this answers is not in the document]
Comment (tpfau) [reply triton/apps/jupyter.rst:296]: My current assumption:
The 'name' parameter in the `installspec` command refers to the kernel name, while displayname is what jupyter will display. I actually have no idea, what the "name" parameter in envkernel does.

[the remark this answers is not in the document]
Comment (rkdarst) [reply triton/apps/jupyter.rst:298]: In envkernel, `name` is the identefiar name of the kernel (the directory name, like `ir` or `imatlab`)

I guess let's ignore this practical problem and take this, we can improve it later when we know more.

File: triton/apps/jupyter.rst [hunk 6]
```diff
@@ -274,20 +274,22 @@ is to use the existing R installations on Triton.
...
            # as r-essentials=X.X.X, where X.X.X is your required R version number
            mamba create -n NAME -c conda-forge r-essentials r-irkernel 
+           # If Mamba doesn't work you can also replace it with conda, but usually mamba is a lot faster
 
         The next steps are the same as building a Kernel, except for activating the environment instead of 
         loading the r-irkernel module, since this module depends on the R version.
         the ``displayname`` will be what will be displayed on jupyter ::
         
           # Use Rscript to install jupyter kernel, you need the environment for this.
+          # You need the jupyterhub/live module to point R at the right place for jupyter
```
Comment (rkdarst) [inline triton/apps/jupyter.rst:292]:
```suggestion
          # You need the Python `jupyter` command so R can know the right place to
          # install the kernel (provided by jupyterhub/live)
```

**Pull request merged**